presto-cli/src/main/java/io/prestosql/cli/Console.java

```diff
@@ -124,6 +124,7 @@ public boolean run()
         try (QueryRunner queryRunner = new QueryRunner(
                 session,
                 clientOptions.debug,
+                clientOptions.timeInMilliseconds,
```

Comment:
a) I think better way to capture this configuration is to put in ClientSession and not passing as boolean flag every where...

b) also, can be easily extnsible to remove boolean and put DateFormat secifier for flexiblity in future(need not change this part in current PR)